<?xml version="1.0"?>

<launch>
    
    <param name="pkg_path" value="$(find assignment_2)"/>
	<param name="planner_path" value="$(find rosplan_planning_system)/common/bin/popf"/>
    <param name="data_path" value="$(find assignment_2)/pddl"/>

    <include file="$(find assignment_2)/launch/rosplanner.launch"/>
    
    <include file="$(find assignment_2)/launch/environment.launch"></include>
    <include file="$(find assignment_2)/launch/gmapping.launch"></include>
    <include file="$(find planning)/launch/move_base.launch"></include>
    <node name="robot_logic" pkg="assignment_2" type="robot_logic.py" output="screen"
        launch-prefix="xterm -e" required="true"/>
    <node name="robot_vision" pkg="assignment_2" type="robot_vision.py" output="screen"/>

</launch>
